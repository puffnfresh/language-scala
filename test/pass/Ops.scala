package scalaz.syntax

trait Ops[A] {
  def self: A
}
