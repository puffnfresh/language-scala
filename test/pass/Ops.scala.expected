Just [Packaging ["scalaz","syntax"],TmplDef (TraitDef "Ops" ["A"] (TemplateBody [TemplateStatDcl (DclDef "self" [] [])]))]